src/main/java/use_case/club_create_post/ClubCreatePostInteractor.java

```diff
@@ -4,7 +4,7 @@
 import entity.user.Club;
 
 /**
- * Club post creation use case interactor.
+ * Club post creation use case interactor. Contains all use cases to do with creating a club.
```

Comment:
This Javadoc is wrong. The use case creates a post, not a club/

Reply:
O whoops, I'll change that!